src/swapnaWJun20/FibonacciSeries.java

```diff
@@ -0,0 +1,28 @@
+package swapnaWJun20;
+
+public class FibonacciSeries {
+	public static void main(String[] args) {
+		FibonacciSeries fibonacciSeries = new FibonacciSeries();
+		fibonacciSeries.calculateFibonacciSeries(5);
+	}
+
+	private void calculateFibonacciSeries(int input) {
+
+		if (input == 0) {
+			System.out.println("0");
+		} else if (input == 1) {
+			System.out.println("0 1");
+		} else {
```

Comment:
The upper If else is not required. This can be optimized in the code. When you give 0, the output should be 1. For input 1, the output should be 1 1 and so on.  The code can start from line 16without the else block.

Reply:
Above chnages has been done in the code. Thanks Anup.